testsuite/features/step_definitions/command_steps.rb

```diff
@@ -960,9 +950,9 @@
 
 When(/^I wait until the package "(.*?)" has been cached on this "(.*?)"$/) do |pkg_name, host|
   node = get_target(host)
-  if host == 'sle_minion'
+  if suse_host?(host)
```

Comment:
Same question, where do you set this variable?

Reply:
and same answer.